v19.2/training/users-and-privileges.md

```diff
@@ -87,6 +87,7 @@ Initially, no users other than `root` have privileges, and root has `ALL` privil
     {% include copy-clipboard.html %}
     ~~~ shell
     $ cockroach sql \
+    --insecure
     --certs-dir=certs \
```

Comment:
Delete this line. `--certs-dir` is ignored when `--insecure` is passed to `cockroach sql`.